src/styles/blocks/card/blockscard/starReviewsPrice.scss

```diff
@@ -0,0 +1,78 @@
+.card__review {
+  display: flex;
+  align-items: flex-end;
+  justify-content: flex-end;
+  max-width: 166px;
+  margin-block: 8px;
+  gap: 17px;
+
+  &-amount {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
```

Comment:
Since the default value of `flex-direction` is `row`, you don't need to explicitly specify it unless you want to change it to `column` or `row-reverse`.